src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.nuspec

```diff
@@ -13,5 +13,6 @@
   </metadata>
   <files>
       <file src="NuGet.exe" target="tools" />
+      <file src="NuGet.pdb" target="tools" />
```

Comment:
Tested with: 
```sh
nuget pack .\src\NuGet.Clients\NuGet.CommandLine\NuGet.CommandLine.nuspec -outputdirectory .\Nupkgs\ -symbols
```
Here is the output:
```
Attempting to build package from 'NuGet.CommandLine.nuspec'.
Successfully created package '.\Nupkgs\NuGet.CommandLine.3.5.0-rc1.nupkg'.
Successfully created package '.\Nupkgs\NuGet.CommandLine.3.5.0-rc1.symbols.nupkg'.
```
The .sumbols package does have .pdb in tools after that.